aiida/cmdline/commands/cmd_code.py

```diff
@@ -341,10 +342,10 @@ def relabel(code, label):
 @options.ALL(help='Include hidden codes.')
 @options.ALL_USERS(help='Include codes from all users.')
 @options.PROJECT(type=click.Choice(VALID_PROJECTIONS.keys()), default=['full_label', 'pk', 'entry_point'])
-@options.RAW()
+@click.option('--format', type=click.Choice(['json', 'yaml', 'raw', 'tabular']), default='raw')
```

Comment:
The name of the option `--format` will be automatically converted to the name of the argument that is passed to the function. So it will be `format`, but you added `output_format` as the function argument. This causes the command to fail (as you can see from the failing test on the CI). So you either make the option `--output-format` or you rename the argument to `format`. The former is better, since `format` is a reserved keyword in Python and it is best not to override it.
```suggestion
@click.option('--output-format', type=click.Choice(['json', 'yaml', 'raw', 'tabular']), default='raw')
```

Reply:
Also, you are removing an existing option here, which is breaking the interface. If we were to go in this direction, it would be best to deprecate the `--raw` option (having it print a deprecation warning when used) and then add the new option.